### PR TITLE
Complete app: Space Travelers' Hub

### PR DESCRIPTION
**Pull request by deyemiobaa**
Built by @deyemiobaa and @YannickZahinda 

Highlights:

- Connect React and Redux.
- Handle events in a React app.
- Write unit tests with React Testing Library.
- Use Redux toolkit to handle async data requests
- Manage the global state of the app using Redux
- Write actions and reducers for booking rockets and joining missions
- Write actions and reducers for canceling rockets and leaving missions
- Render UI correctly

### CODE REVIEW

File: src/components/nav/nav.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      </li>
+      <li>
+        <Link to="/missions">Missions</Link>
+      </li>
+      <li>
+        <Link to="/profile">Profile</Link>
+      </li>
+    </ul>
+  </header>
+);
```
Comment (Meltrust) [inline src/components/nav/nav.js]: According to the project requirements:

image

Therefore:

- [ ] Please, use `<NavLink />` for the links on the navigation bar.
- [ ] Underline the active navigation link, so the user knows where he/she is.

That way, we will meet those requirements 💪

File: src/components/missions/missions.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            </tr>
+          )
+          : (
+            <tr key={mission.id}>
+              <td><h3>{mission.name}</h3></td>
+              <td>{mission.description}</td>
+              <td>{inactiveButton}</td>
+              <td><button id={mission.id} type="button" className="join" onClick={handleClick}>Join mission</button></td>
+            </tr>
+          )))}
```
Comment (Meltrust) [inline src/components/missions/missions.js]: According to the project requirements:

image

Therefore:

- [ ] Please, kindly use react conditional rendering syntax for all your conditional rendering on rockets and missions pages.

## Explanation

```javascript
rocket.reserved && ( //render cancel button )
```

This means "if the rocket is reserved then render the cancel button".  If the statement on the left of the `&&` is evaluated true, the thing on the right happens.

You can also do:

```javascript
!rocket.reserved && ( // Don't render cancel button )
```

That way you can build if/else statements using just the `&&`.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @deyemiobaa,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- NIce job enhancing the profile page when no missions or rockets are joined!! 👏✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi TEAM 👋🏽, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**